props/saber_fett263_buttons.h

```diff
@@ -845,35 +4614,89 @@ SaberFett263Buttons() : PropBase() {}
+  bool menu_ = false; // Edit Mode / Menu System active
+  bool force_quote_ = false; // Quote Player active (in place of force effect)
+  bool rehearse_ = false; // Rehearsal Mode active
+  bool choreo_ = false; // Choreography Mode active
+  uint32_t thrust_begin_millis_; // Thrust timer
+  uint32_t push_begin_millis_; // Push timer
+  uint32_t clash_impact_millis_; // Clash timer
+  uint32_t last_twist_millis_; // Last Twist (to prevent gesture spamming)
+  uint32_t last_push_millis_; // Last Push (to prevent gesture spamming)
+  uint32_t last_blast_millis_; // Last Blast (for Battle Mode Multi-Blast detection)
+  uint32_t saber_off_time_millis_; // Off timer
+  uint32_t last_rotate_millis_; // Last Rotation (to prevent gesture spamming)
+  ClashType clash_type_ = CLASH_NONE;
+  MenuType menu_type_ = MENU_TOP;
+  int menu_top_pos_ = 0; // Top menu dial position
+  int menu_sub_pos_ = 0; // Sub menu dial position
+  TrackMode track_mode_ = PLAYBACK_OFF;
+  int track_num_; // Track Number for Track Player
+  int num_tracks_; // Number of Tracks Found
+  int ignite_time_; // Ignition timer for Edit Mode retraction preview
+  int dial_ = -1; // Menu dial "tick"
+  int sub_dial_; // Sub menu dial "tick"
+  int gesture_num_;
+  float twist_menu_ = M_PI / 4; // Twist Menu sensitivity
+  bool choice_ = false;
+  // Edit Mode selection confirmation 
+  // for True/False control when deleting, disabling/enabling or copying
+  bool off_event_ = false; // Do off event in Edit Mode
+  bool restart_ = false; // Ignite blade after off event in Edit Mode   
+  bool next_event_ = false;
+  // Do next event in Edit Mode, allows an action/wav to complete before 
+  // "next event" begins, for use with choreography and ignition/retraction previews where menu sound
+  // would otherwise be truncated by change in state
+  EditColorMode color_mode_;
+  bool edit_color_ = false; // Color Editing Mode active
+  float hsl_angle_ = 0.0; // HSL angle for Color Editing
+#ifdef FETT263_EDIT_MODE_MENU
+  uint32_t variation_revert_; // Variation revert value
+  Color16 saved_color_;
+  int blade_preview_ = 0; // Blade number for "preview" style
+  int style_num_; // builtin style number for selection in Edit Mode, based on original config
+  int font_num_; // Font number from list_fonts array for use in Edit Mode dial
+  int num_fonts_; // Total number of fonts from list_fonts array
+  int num_presets_; // Total number of builtin styles based on original config
+  int blade_num_; // Active Blade Number for editing
+  int effect_num_; // Effect Arg Number
+  int copy_blade_; // Blade to Copy from
+  int set_num_; // Settings Arg Number
+  int style_revert_; // Original Style Number for Revert
+  int length_revert_; // Original Blade Length for Revert
+  int arg_revert_; // Original Arg vale for Revert
+  // Calculated value for IntArg (Settings Arguments)
+  // used to add or subtract from original saved value by dial
+  int calc_; 
```

Comment:
Please rename calc_ to something more helpful.
Maybe edit_mode_number_ or something?

Reply:
You previously replied to just comment a reminder that you'd use a search and replace for "calc_" after we check in it's noted at line 977.  I'd have to manually replace in 84 locations otherwise.